solr/solrj-streaming/src/java/org/apache/solr/client/solrj/io/eval/Matrix.java

```diff
@@ -61,8 +61,8 @@ public List<String> getRowLabels() {
     return rowLabels;
   }
 
-  public void setRowLabels(List<String> rowLables) {
-    this.rowLabels = rowLables;
+  public void setRowLabels(List<String> rowLabels) {
```

Comment:
this one is okay because we only change the parameter name, not the method signature.